libs/@hashintel/type-editor/src/entity-type-editor/shared/link.tsx

```diff
@@ -0,0 +1,40 @@
+import { HTMLProps, KeyboardEvent, MouseEvent, PropsWithChildren } from "react";
+
+import { useCustomizationSettings } from "../../shared/customization-context";
+
+export const Link = ({
+  children,
+  ...props
+}: PropsWithChildren<HTMLProps<HTMLAnchorElement>>) => {
+  const { onNavigate } = useCustomizationSettings();
```

Comment:
I don't think I understand the point of this component?

Reply:
The type editor is a separate package now, which is also used in blockprotocol.org. It therefore can't use special components like `Link` which is inside `frontend`.

To avoid having simple `a` tags which would cause full page refresh, we need some way of users of the component to handle navigation rather than the package. 

I went for this callback rather than passing in a link component, as it seemed more flexible and less of a type issue. I've renamed it and documented it in https://github.com/hashintel/hash/pull/2117/commits/0827b1749bc6c3777530bd636c5c7ba83dce52bc – happy to take other suggestions as to how to deal with this

Reply:
I think the better way to handle this is to pass a link component to be honest – can we mark that as a follow up? Means we can just use our normal link components (i.e, `next`'s)

Reply:
Yes – I will do this in the next PR I do on the type editor, which will be soon